src/Sets/Zonotope.jl

```diff
@@ -493,6 +493,67 @@ function quadratic_map(Q::Vector{MT}, Z::Zonotope{N}) where {N, MT<:AbstractMatr
     return Zonotope(d, remove_zero_columns(h))
 end
 
+"""
+    bound_intersect_2D(Z::Zonotope, L::Line2D)
+
+Return the support function in the direction [0, 1] of the intersection between
+the given zonotope and line.
+
+### Input
+
+- `Z` -- zonotope
+- `L` -- vertical line 2D
+
+### Output
+
+The support function in the direction [0, 1] of the intersection between the
+given zonotope and line.
+
+### Notes
+
+The given line must be vertical and the algorithm assumes that the intetrsection
```

Comment:
needs update: the algorithm *assumes* that the line is vertical, and *checks* that the intersection is non-empty.